src/common/s3util.py

```diff
@@ -109,6 +109,9 @@ def upload_file_obj(self, stream, key, progress_callback, file_name, config=None
     def get_object_size(self, key):
         try:
             res = self.client.head_object(Bucket=self.bucket_name, Key=key)
+            if res['ContentLength'] == 0:
+                msg = f'File {key} file size is 0.'
```

Comment:
It's a bit weird to have two "file"s in the same sentence. Maybe change it to something like "File size is 0 for {key}"?